src/box/lua/upgrade.lua

```diff
@@ -1227,6 +1227,28 @@ local function upgrade_to_2_10_1()
 end
 --------------------------------------------------------------------------------
 
+--------------------------------------------------------------------------------
+-- Tarantool 2.10.4
+--------------------------------------------------------------------------------
+local function revoke_execute_access_to_lua_function_from_role_public()
+    local _priv = box.space[box.schema.PRIV_ID]
+    if box.func.LUA then
+        local row = _priv:get{PUBLIC, 'function', box.func.LUA.id}
+        if row and bit.band(row.privilege, box.priv.X) ~= 0 then
+            local privilege = bit.bxor(row.privilege, box.priv.X)
+            privilege = bit.bor(privilege, box.priv.R)
+            log.info("revoke execute access to 'lua' function from public role")
```

Comment:
Nit: Function names as case sensetive, so technically we should say `'LUA' function` instead of `'lua' function'` here. Sorry for nits.

Reply:
Thanks, fixes.